### PR TITLE
Clarify wizard prompt asking for file path

### LINKED ISSUE #784: [Wizard] prompt for filename unclear
The prompt  *Enter path to build_tarballs.jl to write to* is unclear. I thought it wanted the directory, but it needed the filename. 
```
Enter path to build_tarballs.jl to write to:
> /home/user/Yggdrasil/L/librsync

SystemError: opening file "/home/user/Yggdrasil/L/librsync": Is a directory
```

### PR DESCRIPTION
**Pull request by giordano**
Fix #784.  @aviks is this any better? :grimacing:

### CODE REVIEW

**Comment by aviks** [comment]
probably not? 😞

**Comment by giordano** [comment]
Any concrete suggestion?

**Comment by giordano** [comment]
Maybe in the true wizard spirit we can check if the path is an existing string and if so create a file called `build_tarballs.jl` in there, otherwise assume it's the path to the file?

**Comment by aviks** [comment]
So yeah, thinking about it, I think the problem here is that the computer knows that the filename it needs is `build_tarballs.jl`, yet aks the user to enter it. Computers should ask the user to enter only information that it does not already know.

So by this logic, the user should only be asked to enter the directory.

If you want the user to enter the filename, say something like 'Enter the full path to a file to save the build recipe. This file is traditionally called build_tarball.jl'

**Comment by giordano** [comment]
@aviks ok, changed to prompt for the path of the directory

File: src/wizard/deploy.jl [hunk 1]
```diff
@@ -240,14 +240,14 @@ function _deploy(state::WizardState)
...
+        directory = nonempty_line_prompt(
             "filename",
-            "Enter path to build_tarballs.jl to write to:";
+            "Enter directory where to write build_tarballs.jl to:";
             ins=state.ins,
             outs=state.outs,
         )
-        mkpath(dirname(abspath(filename)))
-        open(filename, "w") do io
+        mkpath(directory))
```
Comment (staticfloat) [inline src/wizard/deploy.jl:249]:
```suggestion
        mkpath(directory)
```

**Pull request merged**